libs/coin-algorand/src/types.ts

```diff
@@ -0,0 +1,66 @@
+import type {
+  Account,
+  AccountRaw,
+  Operation,
+  OperationRaw,
+  TransactionCommon,
+  TransactionCommonRaw,
+  TransactionStatusCommon,
+  TransactionStatusCommonRaw,
+} from "@ledgerhq/types-live";
+import type { BigNumber } from "bignumber.js";
+
+export const AlgorandOperationTypeEnum = {
+  PAYMENT: 0,
+  ASSET_OPT_IN: 7,
+  ASSET_OPT_OUT: 8,
+  ASSET_TRANSFER: 9,
+};
+
+export type AlgorandResources = {
+  rewards: BigNumber;
+  // Ledger Live only supports a limited list of ASA (defined here https://github.com/LedgerHQ/ledgerjs/blob/master/packages/cryptoassets/src/data/asa.js)
```

Comment:
Outdated link... not super important, but it should just be removed, as it's still changing and will be on private repo `crypto-assets` soon.

Reply:
Comment remove here 132b70f28cc7f8937c815dd83c9e8d62eee73871